packages/now-build-utils/src/fs/download.ts

```diff
@@ -25,7 +25,7 @@ async function downloadFile(file: File, fsPath: string): Promise<FileFsRef> {
     await symlink(target, fsPath);
     return FileFsRef.fromFsPath({ mode, fsPath });
   } else {
-    const stream = file.toStream();
+    const stream = ('toStreamAsync' in file) ? await file.toStreamAsync() : file.toStream();
```

Comment:
You can also try:
```suggestion
    const stream = file instanceof FileFsRef ? await file.toStreamAsync() : file.toStream();
```

If `toStreamAsync` only exist for `FileFsRef`

Reply:
Let's do!